doc/data/messages/d/duplicate-argument-name/related.rst

```diff
@@ -0,0 +1,2 @@
+`Testcases <https://github.com/PyCQA/pylint/blob/master/tests/functional/d/duplicate_argument_name.py>`_
```

Comment:
I understand why `pylint-error` has these in their examples, but I'm a bit scared that these links can easily change if we move tests around.
The Issue Tracker link also doesn't really capture all issues about this message if people use other names in their issues so I think it is not fool-proof.

Reply:
Understandable 👍, I will remove it